CMakeLists.txt

```diff
@@ -15,14 +15,7 @@ message(STATUS "BLAS Libraries: ${BLAS_LIBRARIES}")
 # include_directories(${BLAS_INCLUDE_DIRS})
 
 set(BLA_DEFINITIONS)
-if (BLA_VENDOR MATCHES Intel)
-    if ("$ENV{MKLROOT}" STREQUAL "")
-        message(FATAL_ERROR "MKLROOT is not set. Please source the Intel MKL mklvars.sh file.")
-    endif()
-    find_path(MKL_INCLUDE_DIRS NAMES mkl.h
-              PATHS $ENV{MKLROOT}/include/)
-    message(STATUS "MKL Include: ${MKL_INCLUDE_DIRS}")
-    include_directories(${MKL_INCLUDE_DIRS})
```

Comment:
I am pretty sure this will break travis. Please revert but I am fine if you make setting MKLROOT env variable not mandatory. This variable is set by the mklvars.sh setup script given by MKL Intel package.

Reply:
This doesn't work anyway. Why not include cblas.h in this project?